File: app/src/main/java/com/friday/ar/activities/FridayActivity.java
package com.friday.ar.activities;

import android.os.Bundle;
import android.util.Log;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;


/*
This file is necessarry for analytics, in order to record usage statistics.
 */
public abstract class FridayActivity extends AppCompatActivity {
    private static final String LOGTAG = "FridayActivity";

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.d(LOGTAG, "started activity");
    }

    @Override
    protected void onStop() {
        Log.d(LOGTAG, "stopped activity");
        super.onStop();
    }
}

File: app/src/main/java/com/friday/ar/activities/FridayActivity.kt
package com.friday.ar.activities

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity


/*
This file is necessary for analytics, in order to record usage statistics.
 */
abstract class FridayActivity : AppCompatActivity() {
    companion object {
        const val LOGTAG = "FridayActivity"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Log.d(LOGTAG, "started activity")
    }

    override fun onStop() {
        Log.d(LOGTAG, "stopped activity")
        super.onStop()
    }
}
